jenkins.sh

```diff
@@ -1,104 +1,14 @@
-#!/bin/bash -u
+#!/bin/bash -eu
 
-: ${GOPATH:=${WORKSPACE}/.gopath}
```

Comment:
Nope, the gopath is only needed to build the binary. The dendron binary itself is a static executable.